langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -53,6 +53,11 @@ public type Text xml;
 
 # Returns number of xml items in an xml value.
 #
+# ```ballerina
+# xml books = xml `<book><title>Ballerina</title></book><book><title>Java</title></book>`;
+# books.length() ⇒ 2
```

Comment:
```suggestion
# xml `<book><title>Ballerina</title></book><book><title>Java</title></book>`.length() ⇒ 2
```

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/8bceaabd7a1a2202d2929fee34d1e15e2f2b9b18